tests/test_madlib.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-from madlib_cli.madlib import read_template, parse_template, merge
+from madlib_cli import read_template, parse_template, merge
```

Comment:
`from madlib_cli.madlib import...` should be the correct way to access your functions.

In order for this to be the case, `madlib.py` should be within a directory (module) named `madlib_cli`. Python best practices dictate that scripts should be placed inside of discrete modules. This helps separate them from other files commonly found at the root of a project, like your gitignore, readme, and requirements.txt file.